File: Sort Colors - Leetcode 75/Sort Colors - Leetcode 75.java
public class Solution {
    public void sortColors(int[] nums) {
        int[] counts = new int[3];

        for (int color : nums) {
            counts[color]++;
        }

        int R = counts[0], W = counts[1], B = counts[2];

        for (int i = 0; i < R; i++) {
            nums[i] = 0;
        }
        for (int i = R; i < R + W; i++) {
            nums[i] = 1;
        }
        for (int i = R + W; i < nums.length; i++) {
            nums[i] = 2;
        }
    }
}

File: Sort Colors - Leetcode 75/Sort Colors - Leetcode 75.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        counts = [0, 0, 0]

        for color in nums:
            counts[color] += 1
        
        R, W, B = counts
        
        nums[:R]    = [0] * R
        nums[R:R+W] = [1] * W 
        nums[R+W:]  = [2] * B  